rasa/nlu/featurizers/dense_featurizer/mitie_featurizer.py

```diff
@@ -130,7 +186,7 @@ def features_for_tokens(
             sequence_features.append(feature_extractor.get_feature_vector(token.text))
         sequence_features = np.array(sequence_features)
 
-        sentence_fetaures = self._calculate_sentence_features(
+        sentence_fetaures = self.aggregate_sequence_features(
```

Comment:
Is this change based on changes to `DenseFeaturizer2`? Double-checking I'm not too sure for the reason of this diff.

Reply:
Exactly. It was renamed to `aggregate_sequence_features` in `DenseFeaturizer2`